README.md

```diff
@@ -23,6 +23,8 @@ You can edit the longitude and latitude values in `launch/demo.gps` to change th
 
 Check the Usage section below to learn how to use the position of your robot and a satellite map.
 
+See demo file `launch/demo_utm.launch` for an example of using this plugin in the `UTM frame` mode (see below).
```

Comment:
Thank you for being complete and adding a demo and an updated readme :1st_place_medal: